jetstream/nimbus.py

```diff
@@ -172,34 +171,30 @@ class _FeatureResolver:
     consuming files with leading underscores.
     """
 
-    FEATURE_DEFINITION_REPO = "mozilla/nimbus-shared"
+    FEATURE_DEFINITION_REPO = "https://github.com/mozilla/nimbus-shared"
     FEATURE_PATH = "data/features"
 
     @property
     def data(self) -> Dict[str, Feature]:
         if data := getattr(self, "_data", None):
             return data
-        g = Github()
-        repo = g.get_repo(self.FEATURE_DEFINITION_REPO)
-        specs = repo.get_contents(self.FEATURE_PATH)
-
-        if isinstance(specs, ContentFile):
-            specs = [specs]
-
-        data = {}
-
-        for spec in specs:
-            p = Path(spec.name)
-            slug = p.stem
-            if slug.startswith("_") or p.suffix != ".toml":
-                continue
-            contents = toml.loads(spec.decoded_content.decode("utf-8"))
-            contents["slug"] = slug
-            feature = Feature.from_dict(contents)
-            data[slug] = feature
-
-        self._data = data
-        return self._data
+
+        with TemporaryDirectory() as tmp_dir:
+            Repo.clone_from(self.FEATURE_DEFINITION_REPO, tmp_dir)
+
+            data = {}
+
+            for spec in tmp_dir.rglob(self.FEATURE_PATH + "/**/*.toml"):
```

Comment:
```suggestion
            for spec in tmp_dir.glob(self.FEATURE_PATH + "/**/*.toml"):
```

I think rglob just prepends a **/ which we shouldn't need